### PR TITLE
Add support for animated webp

### PR DESCRIPTION
**Pull request by ShaharHD**
Initial support.

Until this fix, if trying to load an animated webp it will fail on invalid file format.
This is the first version which operates only on the first frame for now.

Still on the TODO for next versions:
- Iterate of Decode & Encode on all frames
- Compose a proper `ImageView` with all the frames (so can be animated properly like a gif image).

Also updated the podspec to 0.2.2

### CODE REVIEW

**Comment by ShaharHD** [comment]
@Yeatse would love your input on this.

**Comment by ShaharHD** [comment]
@Yeatse Some updates:
I've got onevcat/Kingfisher#664 merged and pushed (as `3.7.1`), so we're ready to start working on fully implementing webp animation (just like gif is processed) once this PR is completed.

File: KingfisherWebP/Classes/CGImage+WebP.m [hunk 1]
```diff
@@ -7,52 +7,169 @@
...
-static void ReleaseWebPConfig(void *info, const void *data, size_t size)
-{
-    WebPDecoderConfig* config = (WebPDecoderConfig*)info;
-    WebPFreeDecBuffer(&config->output);
-    free(config);
+#pragma mark - Perfomance Settings
+
+const BOOL useThreads = true; //speed up 23%
+const BOOL bypassFiltering = false; //speed up 11%, cause some banding
+const BOOL noFancyUpsampling = false; //speed down 16%, lose some details
```
Comment (Yeatse) [inline KingfisherWebP/Classes/CGImage+WebP.m]: Maybe we could mark these variables as `static` to restrict their scopes?

**Comment by Yeatse** [comment]
Thank you for your great work! Please review my comment added just now. It will be merged later.

File: KingfisherWebP/Classes/CGImage+WebP.m (hunk 1 above)
Comment (ShaharHD) [inline KingfisherWebP/Classes/CGImage+WebP.m]: Agreed, but re-thinking about it, took a different approach, and put it in the Swift code as is should depend on the method calling it and maybe in the future we need to allow more customization

**Comment by ShaharHD** [comment]
@Yeatse Error 65 - please re-run travis.

**Pull request merged**